content/baremetal/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Baremetal"
+date: 2019-06-18T16:42:20+02:00
+lastmod: 2019-06-18T16:42:20+02:00
+publishdate: 2019-06-18T16:42:20+02:00
+draft: false
+weight: 23
```

Comment:
Weight reflects the category order on the left section. We try to keep this alphabetically ordered, so in this case, Baremetal should be '11' as API is 10... but also you will need to increase 1 on the other categories' `_index.md` files:

* API = 10
* Baremetal = 11
* Certificates = 12 (it was 11)
* etc

Reply:
fixed. I've also added a simple script to fix weights